app/controllers/workflow_repository_controller.rb

```diff
@@ -0,0 +1,38 @@
+class WorkflowRepositoryController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericListMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
+  include Mixins::BreadcrumbsMixin
+
+  menu_section :embedded_workflow
+
+  def self.model
+    ManageIQ::Providers::Workflows::AutomationManager::ConfigurationScriptSource
+  end
+
+  def show_list
+    @title = _("Workflow Reporitories")
+    assert_privileges('embedded_workflow_repository_view')
```

Comment:
@agrare What is the permission for this one?

Reply:
Following the same pattern as above, `embedded_configuration_script_source_view`